lib/cisco_node_utils/cmd_ref/feature.yaml

```diff
@@ -40,6 +40,12 @@ pim:
   get_value: '/^feature pim$/' # TBD pim6
   set_value: 'feature pim'
 
+private_vlan:
```

Comment:
This will need an additional `  _exclude: [N8k]`, since it does not yet have support for `private-vlan`.